CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ should change the heading of the (upcoming) version to include a major version b
 - Added the `idPrefix`, `idSeparator` and `rawErrors` optional props to `FieldProps` since they were missing
 
 ## Dev / docs / playground
+- Migrated latest documentation to Docusaurus, and deploying to GitHub Pages
```

Comment:
Did you also want to mention updating the `README.md` files in all of the packages as well?